### ISSUE TITLE
do we need a config dict for MEDSMaker?

### ISSUE THREAD

**Issue by esheldon** [beckermr/desmedser#16]
The only required field is `medsconf`, and the only optional fields are
```
bitmask_allowed
bounds_buffer_radec
image_dtype
weight_dtype
seg_dtype
bmask_dtype
```
could we get away with keywords?

**Comment by esheldon** [comment]
medsconf is not even propagated anywhere

**Comment by beckermr** [comment]
See my arguments for keeping this on slack (even though it is already gone...).

**Comment by beckermr** [comment]
agreed on this one, but not implemented, moving an issue to esheldon/meds#14
